Libraries/Components/Button.js

```diff
@@ -25,107 +25,114 @@ import type {ColorValue} from '../StyleSheet/StyleSheet';
+
+    @platform android, tv
    */
   nextFocusLeft?: ?number,
 
   /**
-   * TV next focus right (see documentation for the View component).
-   *
-   * @platform android
+    Designates the next view to receive focus when the user navigates right. See the [Android documentation](https://developer.android.com/reference/android/view/View.html#attr_android:nextFocusRight).
+
+    @platform android, tv
    */
   nextFocusRight?: ?number,
 
   /**
-   * TV next focus up (see documentation for the View component).
-   *
-   * @platform android
+    Designates the next view to receive focus when the user navigates up. See the [Android documentation](https://developer.android.com/reference/android/view/View.html#attr_android:nextFocusUp).
+
+    @platform android, tv
    */
   nextFocusUp?: ?number,
 
   /**
-   * Text to display for blindness accessibility features
+    Text to display for blindness accessibility features.
    */
   accessibilityLabel?: ?string,
 
   /**
-   * If true, disable all interactions for this component.
+    If `true`, disable all interactions for this component.
+
+    @default false
    */
   disabled?: ?boolean,
 
   /**
-   * Used to locate this view in end-to-end tests.
+    Used to locate this view in end-to-end tests.
    */
   testID?: ?string,
 |}>;
 
 /**
- * A basic button component that should render nicely on any platform. Supports
- * a minimal level of customization.
- *
- * <center><img src="img/buttonExample.png"></img></center>
- *
- * If this button doesn't look right for your app, you can build your own
- * button using [TouchableOpacity](docs/touchableopacity.html)
- * or [TouchableNativeFeedback](docs/touchablenativefeedback.html).
- * For inspiration, look at the [source code for this button component](https://github.com/facebook/react-native/blob/master/Libraries/Components/Button.js).
- * Or, take a look at the [wide variety of button components built by the community](https://js.coach/react-native?search=button).
- *
- * Example usage:
- *
- * ```
- * import { Button } from 'react-native';
- * ...
- *
- * <Button
- *   onPress={onPressLearnMore}
- *   title="Learn More"
- *   color="#841584"
- *   accessibilityLabel="Learn more about this purple button"
- * />
- * ```
- *
+  A basic button component that should render nicely on any platform. Supports a minimal level of customization.
+
+  If this button doesn't look right for your app, you can build your own button using
+  [TouchableOpacity](touchableopacity) or [TouchableWithoutFeedback](touchablewithoutfeedback).
+  For inspiration, look at the [source code for this button component](https://github.com/facebook/react-native/blob/master/Libraries/Components/Button.js).
+  Or, take a look at the [wide variety of button components built by the community](https://js.coach/?menu%5Bcollections%5D=React%20Native&page=1&query=button).
+
+  Example usage:
+
+  ```
+  import { Button } from 'react-native';
```

Comment:
I noticed that this line isn't being displayed anywhere in the docs but wanted to confirm before getting rid of it. Also the next line with three dots. What do you suggest?

Reply:
Let's assume that the content in the docs is correct and copy it here exactly. So we should remove the "Example usage" line, the `import` etc. This is a general principle as far as I'm concerned: this type of issue will come up fairly often, as a lot of the existing code comments are _similar_ but not identical to the content in the docs. The latter should always "win" unless it's actually incorrect (lists a property that doesn't exist, has typos, etc).